README.md

```diff
@@ -62,6 +62,35 @@ if the exception is not going to be retried, or has failed the final retry.
 | `retryable_exceptions` | `nil`   | Same as for [constant_retry](#constant_retry-options).
 | `fatal_exceptions`     | `[]`    | Same as for [constant_retry](#constant_retry-options).
 
+## Supported backends
+
+Any queue adapter which supports delayed enqueuing (i.e. the `enqueue_at`
+method) will work with ActiveJob::Retry, however some queue backends have
+automatic retrying which should be avoided. The cleanest way to do this is to
+use a `rescue_from` in the jobs for which you're using ActiveJob::Retry. E.g.:
```

Comment:
"using ActiveJob::Retry, so the queue backend never perceives the job as having failed."